clients/cobol-lsp-vscode-extension/package.json

```
@@ -408,15 +408,15 @@
         "cobol-lsp.cpy-manager.endevor-dependencies": {
           "type": "string",
           "enum": [
-            "COBOL LS",
-            "ENDEVOR"
+            "ZOWE",
+            "ENDEVOR_PROCESSOR"
           ],
           "enumDescriptions": [
-            "Retrieve copybooks from mainframe data sets and USS files specified in the COBOL Language Support extension settings",
-            "Retrieve copybooks from locations specified in the Endevor element processor group."
+            "Use ZOWE Explorer to retrieve copybooks from mainframe data sets and USS files specified in the COBOL Language Support extension settings.",
```

based on Zeb's feedback...
```suggestion
            "Use Zowe Explorer to retrieve copybooks from mainframe data sets and USS files specified in the COBOL Language Support extension settings.",
```